### PR TITLE
Removes old crap from rules

### PR DESCRIPTION
**Pull request by Asheboy**
This has Rich's changes from #38 merged in.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,26 +30,24 @@ They must contribute 10p to the biscuit fund.
...
 
 ### Multiple of same value cards
-When in a multiple exemption or redraw (or both), having multiple of the same value card is always considered the highest number. Therefore, a player would require 3 of the same value card to beat 2 of the same valued card. Jokers still trump this rule.
+When in a multiple exemption or redraw (or both), having multiple of the same value card is always considered the highest number. Therefore, a player would require 3 of the same value card to beat 2 of the same valued card. Jokers still trump this rule and reverse jokers make these the worst hands.
 
 ## Draw
-In what turns out to be, the unlikely event of a draw. the two (or more) players must play a face-off. All cards are returned to the dealer, who shuffles the pack and again fans the cards for the 2 (or more) tied players. They compete (lowest card again loses), this is repeated until a loser is decided.
+If two or more players all have the slowest scoring hand, they must draw the same number of cards from the deck again. Any cards drawn in the first round have no bearing on these newly drawn cards. This is repeated until a loser is found.
```
Comment (KenanYusuf) [inline README.md]: > slowest scoring hand

Please fix this.

Comment (Asheboy) [reply README.md]: Fixed, my lord.

**Comment by KenanYusuf** [comment]
👎  I like the new rules, but the level of sarcasm in reply to my review was out of line.

**Comment by jack828** [comment]
👍  LGTM

**Comment by dompainter** [comment]
👍

**Comment by richhastings** [comment]
👍

**Pull request merged**